test/clj_gatling/example.clj

```diff
@@ -3,5 +3,9 @@
 (def test-simu
   {:name "Test simulation"
    :scenarios [{:name "Test scenario"
-                :steps [{:name "Step1" :request (fn [_] true)}
-                        {:name "Step2" :request (fn [_] false)}]}]})
+                :steps [{:name "Step1" :request (fn [_]
+                                                  (Thread/sleep 10)
+                                                  true)}
+                        {:name "Step2" :request (fn [_]
+                                                  (Thread/sleep 10)
+                                                  false)}]}]})
```

Comment:
This goes with the pipeline test, to make sure we can reliably pass when interleaving the test run itself and calling force stop.